base/deprecated.jl

```diff
@@ -1000,4 +1000,20 @@ macro vectorize_2arg(S,f)
 end
 export @vectorize_1arg, @vectorize_2arg
 
+# Deprecate manually vectorized real methods in favor of compact broadcast syntax
+@deprecate real{T<:Real}(x::AbstractArray{T}) real.(x)
+@deprecate real(A::AbstractArray) real.(A)
+@deprecate real(D::Diagonal) real.(D)
+@deprecate real(M::Bidiagonal) real.(M)
+@deprecate real(M::Tridiagonal) real.(M)
+@deprecate real(M::SymTridiagonal) real.(M)
```

Comment:
Good catch! Those deprecations are redundant, the result of mechanical rather than thoughtful work. Simplifying. Thanks!